apps/E2E/src/common/NavigateAppPage.ts

```diff
@@ -315,6 +315,15 @@ class NavigateAppPage extends BasePage {
   private get experimentalTabsPage() {
     return By(HOMEPAGE_EXPERIMENTAL_TABS_BUTTON);
   }
+
+  get _pageName() {
+    // This is an odd name, but it works well with the error message it is used for.
```

Comment:
This comment is a bit confusing. What error message is it used for? Why does it work well?

Reply:
Each spec calls `NavigateAppPage.waitForPageDisplayed()` at the beginning of its file. If this fails, then this error message gets thrown: 
```ts
timeoutMsg: this._pageName + ' did not render correctly. Please see /errorShots for more information.',
```
which reads "The Fluent Tester did not render correctly. Please see /errorShots for more information.".

Reply:
I did make a comment about this in the latest commit.

Reply:
Let's rename it to "FluentTester Base Page", since that's what we're technically checking for with `get _testPage()`.

Then, the comment can simply read "This is the first (main) page displayed after launching FluentTester". This makes more sense and translates better